anndata/tests/test_concatenate.py

```diff
@@ -799,44 +809,42 @@ def test_awkward_does_not_mix(join_type, other):
         concat([adata_a, adata_b], join=join_type)
 
 
-def test_pairwise_concat(axis, array_type):
-    dim_sizes = [[100, 200, 50], [50, 50, 50]]
-    if axis:
-        dim_sizes.reverse()
-    Ms, Ns = dim_sizes
-    dim = ("obs", "var")[axis]
-    alt = ("var", "obs")[axis]
-    dim_attr = f"{dim}p"
-    alt_attr = f"{alt}p"
+def test_pairwise_concat(axis, axis_name, alt_axis_name, array_type):
+    axis_sizes = [[100, 200, 50], [50, 50, 50]]
+    if axis_name == "var":
+        axis_sizes.reverse()
+    Ms, Ns = axis_sizes
+    axis_attr = f"{axis_name}p"
+    alt_attr = f"{alt_axis_name}p"
 
-    def gen_dim_array(m):
+    def gen_axis_array(m):
         return array_type(sparse.random(m, m, format="csr", density=0.1))
 
     adatas = {
         k: AnnData(
             **{
                 "X": sparse.csr_matrix((m, n)),
-                "obsp": {"arr": gen_dim_array(m)},
-                "varp": {"arr": gen_dim_array(n)},
+                "obsp": {"arr": gen_axis_array(m)},
+                "varp": {"arr": gen_axis_array(n)},
             }
         )
         for k, m, n in zip("abc", Ms, Ns)
     }
 
-    w_pairwise = concat(adatas, axis=axis, label="orig", pairwise=True)
-    wo_pairwise = concat(adatas, axis=axis, label="orig", pairwise=False)
+    w_pairwise = concat(adatas, axis=axis_name, label="orig", pairwise=True)
+    wo_pairwise = concat(adatas, axis=axis_name, label="orig", pairwise=False)
```

Comment:
```suggestion
    w_pairwise = concat(adatas, axis=axis, label="orig", pairwise=True)
    wo_pairwise = concat(adatas, axis=axis, label="orig", pairwise=False)
```

I'm not sure I see why we should change this. I think just adding another test which makes sure the two variants behave the same would be better.